### DISCUSSION TITLE
Idea: A setting to disable all routes accept built-in admin routes.

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#6022] [Q&A]
It would be nice to be able to just toggle a setting in the PB built-in admin UI to disable all user defined routes.  This would be useful for doing a series of manual updates without needing to take the server down, and to work on the database in another location, and then to reupload it.

Maybe it's not technically feasible (how would it be done, would a boot event not be fired?) and should be done custom for each project, but it seems pretty useful for when migrations are too much hassle or are too difficult to program.

**Comment by chnfyi** [comment]
A fleeting idea: maybe you can disable the routes you want when you want using `routerUse(middleware)`, an environment variable as a condition and something like `e.request.url` to check the route being requested.
